sdk/storage/azure-storage-file-datalake/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.4.0b1 (2021-05-12)
+**New features**
+- Added support `set_service_properties()`,`get_service_properties()` on `DataLakeServiceClient`
+- Added support for `list_deleted_paths()` on `FileSystemClient`
+
```

Comment:
lets merge https://github.com/Azure/azure-sdk-for-python/pull/18559 and add it here as well!

Reply:
Added